src/seedsigner/models/seed_storage.py

```diff
@@ -58,10 +62,20 @@ def pending_mnemonic(self) -> List[str]:
     def pending_mnemonic_length(self) -> int:
         return len(self._pending_mnemonic)
 
+    
+    @property 
+    def _pending_is_bip39(self) -> bool:
+        return self._pending_seed_type == self.PENDING_SEED_TYPE_BIP39
+        
+        
+    @property
+    def _pending_is_electrum(self) -> bool:
+        return self._pending_seed_type == self.PENDING_SEED_TYPE_ELECTRUM
+
 
-    def init_pending_mnemonic(self, num_words:int = 12, is_electrum:bool = False):
+    def init_pending_mnemonic(self, num_words:int = 12, seed_type: str = None):
```

Comment:
We always specify the intended default value in cases where we heavily expect the default to be the norm.